scripts/generate_go.sh

```diff
@@ -63,6 +63,15 @@ if [ -z $SKIP_BUILD ]; then
   mvn package
 fi
 
+# Clean previously (stale) generated files before regenerating them.
+# Some files are hand-written instead of generated, so preserve those files here.
+find $SDK_DIR/client/v2/common/models/* $SDK_DIR/client/v2/algod/* $SDK_DIR/client/v2/indexer/* \
+  \! -name 'blockRaw.go' \
+  \! -name 'dryrun.go' \
+  \! -name 'account_error_response.go' \
+  \! -name 'shim.go' \
```

Comment:
@algochoi Minor and optional - In my experience, alphabetizing lists like this one simplify long-term maintenance.  If you prefer as is, feel welcomed to resolve.

```suggestion
  \! -name 'account_error_response.go' \
  \! -name 'blockRaw.go' \
  \! -name 'dryrun.go' \
  \! -name 'shim.go' \
```